tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2228,6 +2007,42 @@ public function userGetsDetailsOfTheGroupUsingTheGraphApi(string $user, string $
 		);
 	}
 
+	/**
+	 * @Then /^the JSON data of the response should (not )?contain the user "([^"]*)" in the item 'value'(?:, the user-details should match)?$/
+	 * @Then /^the JSON data of the response should (not )?contain the group "([^"]*)" in the item 'value'(?:, the group-details should match)?$/
+	 *
+	 * @param string $shouldOrNot (not| )
+	 * @param string $userOrGroup
+	 * @param PyStringNode|null $schemaString
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theJsonDataResponseShouldContainDisplayNameAndMatch(
+		string $shouldOrNot,
+		string $userOrGroup,
+		?PyStringNode $schemaString = null
+	): void {
```

Comment:
should it be null in any case?
```suggestion
	 * @param string $userOrGroup
	 * @param PyStringNode $schemaString
	 *
	 * @return void
	 * @throws Exception
	 */
	public function theJsonDataResponseShouldContainDisplayNameAndMatch(
		string $shouldOrNot,
		string $userOrGroup,
		PyStringNode $schemaString
	): void {
```

Reply:
yes it can be null when user or group is not found case in this step.